vhdbuilder/packer/components.json

```diff
@@ -1,15 +1,5 @@
 {
   "ContainerImages": [
-    {
-      "downloadURL": "mcr.microsoft.com/oss/kubernetes/kubernetes-dashboard:*",
-      "amd64OnlyVersions": [],
-      "multiArchVersions": []
-    },
-    {
-      "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
-      "amd64OnlyVersions": [],
-      "multiArchVersions": []
-    },
     {
```

Comment:
you can remove this as well. It's part of the kube-dashboard

Reply:
+1 on metrics-scaper